### ISSUE TITLE
Epic: Game Localization

### ISSUE THREAD

**Issue by OpenTournament** [OpenTournament/OpenTournament#55]
Problem:
As a user, I want to be able to play Open Tournament in my native language.

Solution:
Open Tournament should support localization into other languages.

This is a high level objective and should be considered when developing UI & Audio, and other game elements. Other issues should reference this Issue/Epic/Feature as tasks are clarified and elaborated on.

**Comment by Saibamen** [comment]
IMHO we should implement possibility to translate files without installed Unreal Engine

**Comment by OpenTournament** [comment]
> 
> 
> IMHO we should implement possibility to translate files without installed Unreal Engine

Please clarify, this doesn't make sense as it is.

**Comment by Saibamen** [comment]
Translators should translate Open Tournament with notepad, Qt Linguist, MemoQ, Trados, SDLX, or web online translation tools like XTM Cloud, Crowdin, Transifex.
We can't force them to install Unreal Engine just to translate the game...

**Comment by OpenTournament** [comment]
> Translators should translate Open Tournament with notepad, Qt Linguist, MemoQ, Trados, SDLX, or web online translation tools like XTM Cloud, Crowdin, Transifex.
> We can't force them to install Unreal Engine just to translate the game...

Of course. That isn't needed to do translation, once the engine/project framework is properly set up.
